code/game/objects/items.dm

```diff
@@ -932,10 +932,10 @@ GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/e
 		MO.desc = "Looks like this was \an [src] some time ago."
 		..()
 
-/obj/item/proc/microwave_act(obj/machinery/microwave/microwave_source, mob/microwaver)
+/obj/item/proc/microwave_act(obj/machinery/microwave/microwave_source, mob/microwaver, randomize_pixel_offset)
```

Comment:
I just want you to know that if it wasn't for this linter error I was genuinely gonna speedmerge this